gooddata-sdk/gooddata_sdk/compute/model/execution.py

```diff
@@ -86,10 +98,63 @@ def is_one_dim(self) -> bool:
     def is_two_dim(self) -> bool:
         return len(self.dimensions) == 2
 
+    def _create_value_sort_key(self, sort_key: dict) -> models.SortKey:
+        sort_key_value = sort_key["value"]
+        return models.SortKey(
+            value=models.SortKeyValueValue(
+                data_column_locators=sort_key_value["dataColumnLocators"], direction=sort_key_value["direction"]
+            )
+        )
+
+    def _create_attribute_sort_key(self, sort_key: dict) -> models.SortKey:
+        sort_key_attribute = sort_key["attribute"]
+        # ASC direction is default for Tiger backend.
+        # Don't send it to execution to prevent override of possible default sort label direction.
+        # It has the same meaning as TigerSortDirection.DEFAULT which does not exist.
+        if sort_key_attribute["direction"] == "ASC":
+            return models.SortKey(
+                attribute=models.SortKeyAttributeAttribute(
+                    attribute_identifier=sort_key_attribute["attributeIdentifier"],
+                    sort_type=sort_key_attribute["sortType"],
+                )
+            )
+        return models.SortKey(
+            attribute=models.SortKeyAttributeAttribute(
+                attribute_identifier=sort_key_attribute["attributeIdentifier"],
+                direction=sort_key_attribute["direction"],
+                sort_type=sort_key_attribute["sortType"],
+            )
+        )
+
+    def _create_sort_keys(self, sort_keys: list[dict]) -> list[models.SortKey]:
+        api_sort_keys: list[models.SortKey] = []
+        for sort_key in sort_keys:
+            if "value" == list(sort_key.keys())[0]:
+                api_sort_keys.append(self._create_value_sort_key(sort_key))
+            elif "attribute" == list(sort_key.keys())[0]:
+                api_sort_keys.append(self._create_attribute_sort_key(sort_key))
+            else:
+                continue
```

Comment:
Is order really important in `sort_key.keys()`? If not, please do the following:

```python
value = sort_key.get("value")
attribute = sort_key.get("attribute")
if value is not None:
    api_sort_keys.append(self._create_value_sort_key(sort_key))
if attribute is not None:
    api_sort_keys.append(self. _create_attribute_sort_key(sort_key))
```